R/truncnorm.R

```diff
@@ -31,6 +33,9 @@ my_etruncnorm= function(a,b,mean=0,sd=1){
   toobig = max_alphabeta<(-30)
   toobig[is.na(toobig)]=FALSE
   tmp[toobig] = max_ab[toobig]
+
+  sdzero = sd == 0
+  tmp[sdzero] = mean[sdzero]
```

Comment:
as sd-> 0 does the expectation of the truncated normal tend to the mean? I don't think so....